src/liballoc/collections/linked_list.rs

```diff
@@ -878,6 +878,55 @@ impl<T> LinkedList<T> {
         unsafe { self.split_off_after_node(split_node, at) }
     }
 
+    /// Removes the element at the given index and returns it.
+    ///
+    /// # Panics
+    /// Panics if at >= len
```

Comment:
This should mean that we return `T`, not an `Option<T>`, as we're always in bounds. (It is nice to do just T, I think, since that matches Vec::remove).